pkgs/top-level/all-packages.nix

```diff
@@ -7010,9 +7010,8 @@ with pkgs;
   cargo-inspect = callPackage ../development/tools/rust/cargo-inspect {
     inherit (darwin.apple_sdk.frameworks) Security;
   };
-  cargo-lambda = callPackage ../development/tools/rust/cargo-lambda {
+  cargo-lambda = callPackage ../by-name/ca/cargo-lambda/package.nix {
```

Comment:
AFAIK, `by-name` packages do not need to be specified in `all-packages.nix`...

Reply:
That is correct, but see the Zig override right below. As 24.11 is using an older version of cargo-lambda than unstable, and the zig override to zig_0_12 was only set in 24.11 but not in unstable, I assumed it was there for a reason.

Reply:
Also see https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#changing-implicit-attribute-defaults

Reply:
Ah, ... Sorry, I am at Coffee :coffee: number 1, that's too few to do reviews apparently. :laughing: